servers/src/test/java/tachyon/worker/block/allocator/AllocatorFactoryTest.java

```diff
@@ -50,19 +54,29 @@ public void before() throws IOException {
   
   @Test
   public void createGreedyAllocatorTest() {
-    Allocator allocator = AllocatorFactory.create(AllocatorType.GREEDY, mManagerView);
+    mTachyonConf.set(Constants.WORKER_ALLOCATE_STRATEGY, GreedyAllocator.class.getName());
+    Allocator allocator = Allocator.Factory.createAllocator(mTachyonConf, mManagerView);
     Assert.assertTrue(allocator instanceof GreedyAllocator);
   }
   
   @Test
   public void createMaxFreeAllocatorTest() {
-    Allocator allocator = AllocatorFactory.create(AllocatorType.MAX_FREE, mManagerView);
+    mTachyonConf.set(Constants.WORKER_ALLOCATE_STRATEGY, MaxFreeAllocator.class.getName());
+    Allocator allocator = Allocator.Factory.createAllocator(mTachyonConf, mManagerView);
     Assert.assertTrue(allocator instanceof MaxFreeAllocator);
   }
-  
+
+  @Test
+  public void createRoundRobinAllocatorTest() {
+    mTachyonConf.set(Constants.WORKER_ALLOCATE_STRATEGY, RoundRobinAllocator.class.getName());
+    Allocator allocator = Allocator.Factory.createAllocator(mTachyonConf, mManagerView);
+    Assert.assertTrue(allocator instanceof RoundRobinAllocator);
+  }
+
   @Test
-  public void createDefaultAllocatorTypeTest() {
-    Allocator allocator = AllocatorFactory.create(AllocatorType.DEFAULT, mManagerView);
+  public void createDefaultAllocatorTest() {
+    TachyonConf conf = new TachyonConf();
```

Comment:
I think it needs to be a new instance here, to test the default allocator.

Maybe leave a short comment about it?